addons/mail/static/src/core/common/chat_window.xml

```diff
@@ -17,7 +17,7 @@
                 <div class="d-flex text-truncate">
                     <Dropdown position="'left-start'" onStateChanged="isOpen => this.onActionsMenuStateChanged(isOpen)" menuClass="'d-flex flex-column py-0'">
                         <button
-                            class="o-mail-ChatWindow-command btn rounded-0 d-flex align-items-center px-1 py-2 m-0 opacity-75 opacity-100-hover w-100"
+                            class="o-mail-ChatWindow-command btn rounded-3 d-flex align-items-center px-1 py-2 m-0 opacity-75 opacity-100-hover w-100"
```

Comment:
It looks better with `rounded-top-3 rounded-bottom-0`, so the bottom matches separation between header and content:

<img width="567" alt="Screenshot 2024-07-10 at 16 41 56" src="https://github.com/odoo/odoo/assets/6569390/df499b63-bea0-415e-a354-9152de34cb5e">